lapack.go

```diff
@@ -11,22 +11,57 @@ const None = 'N'
 type Job byte
```

Comment:
I wasn't sure what they do, as they were there before I started editing and they are not used in gonum/native. I did what anyone does and googled it. gonum/lapack (and forks) is 4 of the top 7 results.

It seems as if Job does not mean something consistent. Ddisna, for example, has 3 choices for the Job char that are completely unrelated to the ones that are there now. I'd like to leave these uncommented for now. We can change the OpenBLAS build and delete them.